hack/verify-gofmt.sh

```diff
@@ -23,10 +23,15 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
-kube::golang::verify_go_version
-
 cd "${KUBE_ROOT}"
 
+# Prefer bazel's gofmt.
+gofmt="external/io_bazel_rules_go_toolchain/bin/gofmt"
```

Comment:
I think rules_go can expose this more nicely along with the raw go tool, but until that happens this is about as good as we're going to do.